### PR TITLE
Parameters extended for bug report

### PR DESCRIPTION
**Pull request by csengerszabo**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Added "Current workaround", "Affected user persona", "Business goal to be improved" and "Metric to be improved" as bug report parameter requirements
Standardised existing parameters with other repositories

**Which issue(s) this PR fixes** *: -

**Special notes for your reviewer**: -

**Does this PR introduce a user-facing change?**: -

```release-note
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -4,24 +4,29 @@ about: Report a bug related to KubeOne
...
+- Operating system you're deploying on:
 
-**What is the expected behavior**:
+### Current workaround
 
-**How to reproduce the issue**:
+### Affected user persona
 
-**Anything else we need to know?**
+### Business goal to be improved
```
[the remark this answers is not in the document]
Comment (csengerszabo) [reply .github/ISSUE_TEMPLATE/bug-report.md:30]: Referring this: https://github.com/kubermatic/kubermatic/pull/9517#discussion_r840555192

**Comment by xmudrii** [comment]
Suppressed by #1943.

**Pull request closed without merging**